hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileBlock.java

```diff
@@ -840,6 +841,16 @@ public boolean isSharedMem() {
     /** Meta data that holds information about the hfileblock**/
     private HFileContext fileContext;
 
+    void beforeShipped() {
```

Comment:
Why not making it ShipperListener. That would be best

Reply:
done it